<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIFilterDialog</class>
 <widget class="QDialog" name="UIFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>361</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="edit_Class">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>163</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>精确查找</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>5</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>范围筛选</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>姓名</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Class">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>112</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>按班级</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>20</width>
      <height>290</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>340</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_id">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>学号/工号</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_College">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>按学院</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Major">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_College">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_Find">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Major">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>按专业</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>20</width>
      <height>290</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">font-size: 12px</string>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">font-size: 12px</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>checkBox_College</tabstop>
  <tabstop>edit_College</tabstop>
  <tabstop>checkBox_Major</tabstop>
  <tabstop>edit_Major</tabstop>
  <tabstop>checkBox_Class</tabstop>
  <tabstop>edit_Class</tabstop>
  <tabstop>radioButton_name</tabstop>
  <tabstop>radioButton_id</tabstop>
  <tabstop>edit_Find</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
